Libraries/AppState/AppState.js

```diff
@@ -86,9 +86,10 @@ class AppState extends NativeEventEmitter {
     // TODO: getCurrentAppState callback seems to be called at a really late stage
     // after app launch. Trying to get currentState when mounting App component
     // will likely to have the initial value here.
-    // Initialize to 'active' instead of null.
-    this.currentState = 'active';
-    
+    // If RCTAppState implements initialAppStateInitialize, use that. Otherwise,
+    // use 'active'.
+    this.currentState = RCTAppState.initialAppState || 'active';
```

Comment:
I think we need to keep it until we export a `initialAppState` constant on Android.